micro_sam/evaluation/automatic_mask_generation.py

```diff
@@ -0,0 +1,180 @@
+import os
+import json
+import argparse
+import numpy as np
+import pandas as pd
+from tqdm import tqdm
+from glob import glob
+import imageio.v2 as imageio
+from elf.evaluation import mean_segmentation_accuracy
+
+from micro_sam.util import precompute_image_embeddings
+from micro_sam.instance_segmentation import AutomaticMaskGenerator
+
+from .inference import get_predictor
+
+
+def remove_bg_as_biggest_element(image, masks):
```

Comment:
Please remove this and use https://github.com/computational-cell-analytics/micro-sam/blob/master/micro_sam/instance_segmentation.py#L53 instead.